src/apibuilder-cli/file_tracker.rb

```diff
@@ -0,0 +1,72 @@
+module ApibuilderCli
+
+  class FileTracker
+
+  	def FileTracker.default_path
+  		"#{ApibuilderCli::Config::APIBUILDER_LOCAL_DIR}/tracked_files"
```

Comment:
should we make this `.tracked_files` ?

Reply:
I was thinking about that. I was taking a page from the git files, where top-level files/directories are '.' prefixed (`.gitignore`, `.git` dir), but files inside those aren't (`.git/config`). The main concern I was thinking about is that there might be a name collision with scaffolding source files that are dropped into the `.apibuilder` directory (i.e. if a generator produces a scaffolding file named `tracked_files`). So this is a decent argument for the prefix.